examples/docker-compose/.env.example

```diff
@@ -28,8 +28,14 @@ NEO4J_WEB_PORT=7474
 
 # BloodHound Configuration Parameters
 
-# Uncomment if you want to disable cypher quality control
-#bhe_disable_cypher_qc=false
+# Uncomment if you want to disable cypher complexity limit
+#bhe_disable_cypher_complexity_limit=true
```

Comment:
can the comment here be more descriptive?  as a user im not sure what im disabling when i set this flag to true. 

i also noticed that there is another .env.example file at the root of the bhce project.  do these flags need to be specified there as well?

Reply:
That .env.example is available for dev use, but I'm not sure if anyone typically uses it since you can modify your own config file instead. Probably not necessary to update with these flags as a result.

As for being more descriptive, I'm fairly confident that the users that want to disable this will know what is meant. They would have gotten errors saying the complexity was too high, and this gives them a flag to disable that if they want to.

We're happy to hear suggestions for a better comment, but we'll likely want to refine documentation of our environment variables at some point separately for more clarity